lesson-8/Ioan-Juglea/Classes Homework.py

```diff
@@ -0,0 +1,52 @@
+class Produs(object):
```

Comment:
Este recomandat sa numim toate variabilele in engleza, asa facem codul accesibil pentru un numar mai mare de programatori